psi4/src/psi4/libdpd/T3_AAA.cc

```diff
@@ -167,7 +167,7 @@ void DPD::T3_AAA(double ***W1, int nirreps, int I, int Gi, int J, int Gj, int K,
         W2[Gab] = dpd_block_matrix(F->params->coltot[Gab], virtpi[Gc]);
 
         if (F->params->coltot[Gab] && virtpi[Gc]) {
-            ::memset(W1[Gab][0], 0, F->params->coltot[Gab] * virtpi[Gc] * sizeof(double));
+            ::memset(W1[Gab][0], 0, sizeof(double) * F->params->coltot[Gab] * virtpi[Gc]);
```

Comment:
I don't understand why this swap would make a difference?

Reply:
the hypothesis is that lgtm works from language rules and implicit casts multiplication left-to-right, rather than compiler practice, which doesn't care which position the big type is at. we'll have to merge one of these to see how lgtm responds.